mtgjson5/providers/tcgplayer.py

```diff
@@ -250,6 +281,43 @@ def get_tcgplayer_sku_data(group_id_and_name: Tuple[str, str]) -> List[Dict[str,
     return magic_set_product_data
 
 
+def get_tcgplayer_sealed_data(group_id: Optional[Any]) -> List[Dict[str, Any]]:
+    """
+    Finds all sealed product for a given group
+    :param group_id: group id for the set to get data for
+    :return: sealed product data with extended fields
+    """
+    magic_set_sealed_data = []
+    api_offset = 0
+
+    while True:
+        api_response = TCGPlayerProvider().download(
+            "https://api.tcgplayer.com/catalog/products",
+            {
+                "offset": str(api_offset),
+                "limit": 100,
+                "categoryId": 1,
+                "getExtendedFields": True,
```

Comment:
what is this needed for?

Reply:
It is unnecessary for now but could be used to add a content description field that TCG stores

Reply:
let's add it when needed then? the returned data seems either missing or useless, like

```json
            "extendedData": [
                {
                    "name": "OracleText",
                    "displayName": "Rules Text Contains",
                    "value": "Booster box for 10th Edition."
                }
            ]
```

Reply:
I agree with not adding un-used code